common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -597,12 +597,7 @@ def editable_metadata_fields(self):
         possible_sub_ids = [self.sub, self.youtube_id_1_0] + get_html5_ids(self.html5_sources)
         for sub_id in possible_sub_ids:
             try:
-                get_transcript_for_video(
-                    self.location,
-                    subs_id=sub_id,
-                    file_name=sub_id,
-                    language=u'en'
-                )
+                _, sub_id, _ = get_transcript(self, lang=u'en', output_format=Transcript.TXT)
```

Comment:
Unknown what impact this will have, since the output from contentstore transcripts for `sub_id` (filename) will be formatted like `en_subname.txt` instead of `subname`.

Reply:
It works for transcripts in the Demo course, as far as I've been able to test.